src/dopameme/constants.py

```diff
@@ -1,133 +1,143 @@
-    'smol',
-    'snuggly',
-    'sweet',
-    'teensy',
-    'tiny',
+    "adorable",
+    "aegyo",
+    "baby",
+    "charming",
+    "cute",
+    "cutest",
+    "darling",
+    "kawaii",
+    "little",
+    "pygmy",
+    "small",
+    "smol",
+    "smooshy",
+    "snuggly",
+    "softest",
+    "sweet",
+    "teensy",
+    "tiny",
 )
 """Search term - adjectives."""
 
 NOUNS = (
-    'alpaca',
-    'axolotl',
-    'bear',
-    'bird',
-    'birdy',
-    'bunny',
-    'butterfly',
-    'cat',
-    'deer',
-    'doe',
-    'dog',
-    'doggie',
-    'dolphin',
-    'duck',
-    'duckling',
-    'echidna',
-    'fawn',
-    'frog',
-    'hampster',
-    'harp seal',
-    'hedgehog',
-    'junco',
-    'kitten',
-    'kitty',
-    'koala',
-    'newt',
-    'owl',
-    'pangolin',
-    'piggy',
-    'pony',
-    'pupper',
-    'puppy',
-    'quokka',
-    'rabbit',
-    'raccoon',
-    'seal',
-    'sloth',
-    'squirrel',
-    'toad',
-    'tortoise',
-    'turtle',
-    'wolf pup',
-    'wolf',
+    "alpaca",
+    "axolotl",
+    "bear",
+    "bird",
+    "birdy",
+    "bunny",
+    "butterfly",
+    "cat",
+    "deer",
+    "doe",
+    "dog",
+    "doggie",
+    "dolphin",
+    "duck",
+    "duckling",
+    "echidna",
+    "fawn",
+    "frog",
+    "froggy",
+    "hampster",
+    "harp seal",
+    "hedgehog",
+    "junco",
+    "kitten",
+    "kitty",
+    "koala",
+    "lion",
+    "llama",
+    "narwhal",
+    "newt",
+    "owl",
+    "ocelot",
+    "pangolin",
+    "piggy",
+    "pony",
+    "pupper",
+    "puppy",
+    "quokka",
+    "rabbit",
+    "raccoon",
+    "seal",
+    "sloth",
+    "snail",
+    "squirrel",
+    "tiger",
+    "toad",
+    "tortoise",
+    "turtle",
+    "wolf pup",
+    "wolf",
 )
 """Search term - nouns."""
 
 MODIFIERS = (
-    'cuddling',
-    'wearing a hat',
-    'in a house',
-    'playing',
-    'sleeping',
-    'wearing onesie',
-    'in a cup',
-    'on a hill',
-    'purring',
-    'whimpering',
+    "cuddling",
+    "wearing a hat",
+    "in a house",
+    "playing",
+    "sleeping",
+    "wearing onesie",
+    "in a cup",
+    "on a hill",
+    "purring",
+    "whimpering",
 )
 """Optional search terms - modifiers."""
 
 EXCLUDE = (
-    '-disgusting',
-    '-gross',
-    '-icky',
-    '-poop',
-    '-porn',
-    '-sexy',
-    '-shit',
-    '-ugly',
-    '-xxx',
-    '-rotten',
-    '-horrible',
+    "-disgusting",
+    "-gross",
+    "-icky",
+    "-poop",
+    "-porn",
+    "-sexy",
+    "-shit",
+    "-ugly",
+    "-xxx",
+    "-rotten",
+    "-horrible",
 )
 """Terms to exclude from searches"""
 
 # Strings that shouldn't appear in any URLs:
 URL_EXCLUDE = (
-    '?show_error=true',
-    'amazon',
-    'etsy',
-    'google.com/search',
-    'item',
-    'product',
-    'redtube',
-    'teepublic',
-    'tube8',
-    'xnxx',
-    'xxx',
-    'youporn',
-    'zillow',
+    "?show_error=true",
+    "amazon",
+    "etsy",
+    "goldporntube",
+    "google.com/search",
+    "item",
+    "pornhub",
```

Comment:
I'm gonna hope this and the following are preemptive!

Reply:
I added a few that came up one or twice in thousands of test runs. I want cute floofy bunnies not playboy bunnies!